setup.py

```diff
@@ -1,33 +1,32 @@
-import pathlib
-
 from setuptools import setup
 
-import versioneer
-
-here = pathlib.Path(__file__).parent.resolve()
-long_description = open("./fideslog/sdk/python/README.md").read()
+from versioneer import get_cmdclass, get_version
 
-# Requirements
-install_requires = (
-    open("fideslog/sdk/python/requirements.txt").read().strip().split("\n")
-)
-dev_requires = open("dev-requirements.txt").read().strip().split("\n")
+api_requires = open("./fideslog/api/requirements.txt").read().strip().split("\n")
+sdk_requires = open("./fideslog/sdk/python/requirements.txt").read().strip().split("\n")
+dev_requires = open("./dev-requirements.txt").read().strip().split("\n")
 
 setup(
     name="fideslog",
-    version=versioneer.get_version(),
-    cmdclass=versioneer.get_cmdclass(),
-    description="SDK for interacting with fideslog",
-    long_description=long_description,
+    version=get_version(),
+    cmdclass=get_cmdclass(),
+    description="The fideslog analytics collection mechanism",
+    long_description=open("./README.md").read(),
     long_description_content_type="text/markdown",
     url="https://github.com/ethyca/fideslog",
     python_requires=">=3.8, <4",
-    package_dir={"fideslog": "fideslog"},
-    packages=["fideslog.sdk.python"],
+    package_dir={
```

Comment:
This also worked as expected. The docker image contained both the `sdk` and `api` directory.